src/Sylius/Bundle/WebBundle/Resources/translations/messages.en.yml

```diff
@@ -673,6 +675,104 @@ sylius:
         taxes: Taxes
         total: Total
         update_header: Editing order
+        customer: customer
```

Comment:
Why do you have so many new translations not related to reviews? Do we need them?